src/app/clusters/thermostat-server/thermostat-server-suggestions.cpp

```diff
@@ -38,26 +39,31 @@ namespace Thermostat {
 
 extern ThermostatAttrAccess gThermostatAttrAccess;
 
-bool emberAfThermostatClusterAddThermostatSuggestionCallback(CommandHandler * commandObj, const ConcreteCommandPath & commandPath,
-                                                             const Commands::AddThermostatSuggestion::DecodableType & commandData)
+bool AddThermostatSuggestion(CommandHandler * commandObj, const ConcreteCommandPath & commandPath,
+                             const Commands::AddThermostatSuggestion::DecodableType & commandData)
 {
 
     // If time is not synced, return INVALID_IN_STATE in the AddThermostatSuggestionResponse.
-    uint32_t currentTimestamp = 0;
-    if (System::Clock::GetClock_MatterEpochS(currentTimestamp) != CHIP_NO_ERROR)
+    uint32_t currentMatterEpochTimestampInSeconds = 0;
+    if (System::Clock::GetClock_MatterEpochS(currentMatterEpochTimestampInSeconds) != CHIP_NO_ERROR)
     {
         commandObj->AddStatus(commandPath, Status::InvalidInState);
         return true;
     }
 
-    auto delegate = GetDelegate(commandPath.mEndpointId);
+    ChipLogError(Zcl, "current timestamp is %u", currentMatterEpochTimestampInSeconds);
```

Comment:
This log should not be here, or certainly should not be an error log.  But best if it were not here...

Reply:
done.